docs/initial-set-up/installation-and-basic-set-up.md

```diff
@@ -4,11 +4,11 @@ Before reading further, please be aware that much of the information contained h
 
 ## Prerequisites
 
-Before exploring the installation of CiviCRM, ensure you have read the chapter 'hosting' to confirm whether or not your host can support it. 
+Before exploring the installation of CiviCRM, ensure you have read the chapter [hosting](/planning/hosting.md)to confirm whether or not your host can support it. 
 
 CiviCRM must be installed on a computer that has been configured with a web server (such as Apache or ngnx), PHP and MySQL. Some people prefer to try out CiviCRM on their own local computer before installing it on a dedicated web-server. If you are doing this and don't have the prerequisites just mentioned, you can download packages from the Internet such as WAMP, XAMPP, MAMPP and LAMP, which will quickly install an Apache web server, PHP and MySQL. (The first two packages are for Windows and the second two are for the Macintosh and Linux respectively).
 
-It is important to consider where you host your CiviCRM instance. You may want to discuss this with your System Administartor and review the [Hosting information](https://docs.civicrm.org/sysadmin/en/latest/planning/hosting/) in the System Admin guide.
+It is important to consider where you host your CiviCRM instance. You may want to discuss this with your System Administartor and review the [Hosting information](/planning/hosting.md) in the System Admin guide.
```

Comment:
@AnneDru can you revert this and the System Status changes please. The hosting documentation is in the sysadmin guide which is a different guide to this one.